src/components/add-token-button/add-token-button.tsx

```diff
@@ -0,0 +1,64 @@
+import { useAddAssetToWallet } from "../../hooks/use-add-asset-to-wallet";
+
+export const AddTokenButtonLink = ({
+  address,
+  symbol,
+  decimals,
+  image,
+}: {
+  address: string;
+  symbol: string;
+  decimals: number;
+  image: string;
+}) => {
+  const { add, addSupported } = useAddAssetToWallet(
+    address,
+    symbol,
+    decimals,
+    image
+  );
+  if (!addSupported) {
+    return null;
+  }
+  return (
+    <button className="add-token-button button-link" onClick={add}>
+      Click here to add to wallet
```

Comment:
Could we make this a translation.
and make the copy = "Show this token in your Ethereum wallet"